docs/src/layouts/MainLayout.vue

```diff
@@ -161,4 +176,75 @@ $adjust-header-viewport: 860px;
   z-index: 5; // ensures it's lower than header but higher than components page filter section
 }
 
+@function generateRandomStars($number-of-stars) {
+  $value: '#{random(7000)}px #{random(7000)}px #{$lp-primary}';
+  @for $i from 1 through $number-of-stars {
+    $value: '#{$value}, #{random(7000)}px #{random(7000)}px #{$lp-primary}';
+  }
+  @return unquote($value);
+}
+
+$shadows-small: generateRandomStars(900);
+$shadows-medium: generateRandomStars(600);
+$shadows-big: generateRandomStars(500);
+
+#small-stars {
+  width: 1px;
+  height: 1px;
+  background: transparent;
+  box-shadow: $shadows-small;
+  animation: animateStar 50s linear infinite;
```

Comment:
For some reason, seems like the smaller stars "stutter" around, do you see it too?

Reply:
I think maybe there's too many of them and they were moving too fast. I have reduced them and increased their speed too. I think it makes them a little more regular than before. We may have to try other values.